src/Plugins/DBFTPlugin/Consensus/ConsensusService.Check.cs

```diff
@@ -60,7 +60,6 @@ private void CheckCommits()
             if (context.CommitPayloads.Count(p => context.GetMessage(p)?.ViewNumber == context.ViewNumber) >= context.M && context.TransactionHashes.All(p => context.Transactions.ContainsKey(p)))
             {
                 block_received_index = context.Block.Index;
-                block_received_time = TimeProvider.Current.UtcNow;
                 Block block = context.CreateBlock();
```

Comment:
What's happend with this variable?

Reply:
It's removed because it's not used after we implement https://github.com/neo-project/neo/pull/3622#discussion_r2039247568.